back/tests/test_arango_manager.py

```diff
@@ -10,23 +10,28 @@
     get_works_in_rectangle,
     delete_collection_by_name,
     create_collection_by_name,
+    setup_collection,
 )
 import pytest
 
 test_doc_id_1 = None
 test_doc_id_2 = None
 test_doc_id_3 = None
 original_number_of_works = None
+test_collection = "test"
 
 
-@pytest.fixture(scope="session", autouse=True)
+# The scope module is define to use the setup database function for all tests
+@pytest.fixture(scope="module", autouse=True)
```

Comment:
Why was this change required ?

Reply:
I changed it because I identified the session scope as a mistake, because we weren't able to request the arangodatabase with our setup, now the setup procedure  is applied before any test in the test_arango_manager file.